lib/scripts/utils/languages.js

```diff
@@ -0,0 +1,103 @@
+export var bingLanguages = {
```

Comment:
I did think of that. But supported languages don't change that often. So we can manually set them.